linter/whitelists/default.txt

```diff
@@ -20,4 +21,119 @@ TinyPNG
 trilobit
 Dalley
 PhotoSwipe
-Slack
\ No newline at end of file
+Slack
+Catalog
+Manager
+Backend
+Bing
+Bundle
+Bundles
+Changelanguage
+ChangeLanguage
+Twig
+Tweets
+Twitter
+Template
+Templates
+Templating
+Themes
+Theme
+Usability
+WhatsApp
+Widget
+Widgets
+Wizard
+backend
```

Comment:
`backend` is wrong in my opinion. In German, I would use `Backend` and in English it's `back end` or `back-end`. But never `backend`.